OrbitService/LinuxUtils.cpp

```diff
@@ -166,4 +176,24 @@ ErrorMessageOr<std::string> GetExecutablePath(int32_t pid) {
   return std::string(buffer, length);
 }
 
+outcome::result<std::string> FileToString(
+    const std::filesystem::path& file_name) {
+  std::ifstream file_stream(file_name);
+  if (file_stream.fail()) {
+    return outcome::failure(static_cast<std::errc>(errno));
+  }
+  return outcome::success(
+      std::string{std::istreambuf_iterator<char>{file_stream},
+                  std::istreambuf_iterator<char>{}});
+}
```

Comment:
please add tests for this and other methods in this file.

Reply:
I also wonder if this should be moved to base - seems pretty generic.

Reply:
Like I said in the PR description, I wanted to skip refactoring and tests in this PR. But I hear you and I can them in this PR. I will add a 4th commit for that